Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```

Comment:
Using gem `mysql2` instead of sqlite3 for your project
About configuration, refs: https://edgeguides.rubyonrails.org/configuring.html#configuring-a-mysql-or-mariadb-database
Pre-condition: installed MySQL DB on computer